entrypoint-4.2.x.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
```

Comment:
Instead of creating a new file that has the same contents, I would say just create a symlink for `entrypoint-4.2.x.sh` to point to the actual file `entrypoint-4.1.x.sh` since there are no differences.

Reply:
To fix this, I believe it should be:
```
rm entrypoint-4.2.x.sh
ln -s entrypoint-4.1.x.sh entrypoint-4.2.x.sh
```